### PR TITLE
Yashu

### PR DESCRIPTION
**Pull request by yashubari**

### CODE REVIEW

**Review by Snehal-113 — CHANGES_REQUESTED** [review]
The index.js file must be in src folder

**Review by shivnanda6272 — COMMENTED** [review]
Q6 :-  state IS NULL

**Comment by shivnanda6272** [comment]
> Q6 :- state IS NULL

File: Assignments/MYSQL/Yashasvi/YashasviDB1.sql [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+#Q4. Display all the fields from products table.
+SELECT*FROM products;
+
+#Q5. Display the customer name,phone number,city,country of all the customers.
+SELECT customerName,phone,city,country FROM customers;
+
+#Q6. Display the details of customers where the state is null.
+SELECT customerName,customerName
+FROM customers
+WHERE state=NULL;
```
Comment (Snehal-113) [inline Assignments/MYSQL/Yashasvi/YashasviDB1.sql]: state is null

**Review by Snehal-113 — CHANGES_REQUESTED** [review]

**Review by Snehal-113 — APPROVED** [review]

**Pull request merged**